core/src/BibleEngine.class.ts

```diff
@@ -867,6 +875,10 @@ export class BibleEngine {
 
         if (state.recursionLevel === 0) {
             // we are at the end of the root method => persist everything
+            const { osisId } = state.book;
+            if (osisId === 'Ps' || osisId === 'Zech') {
```

Comment:
i guess this is temporary code?

anyway it feels like this code should be on the side of the importer / plugin